doc/tutorials/using_xenomai_as_uos.rst

```diff
@@ -0,0 +1,97 @@
+.. _using_xenomai_as_uos:
+
+Using Xenomai as User OS (Real-Time VM)
```

Comment:
Actually, wondering if it wouldn't be better to say "User VM OS". @deb-intel @dbkinder thoughts? (this can be fixed in a later PR if you're going to have a later pass at the doc anyway.